src/azure-cli/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -472,6 +485,12 @@ def _build_storage_profile():
     }
     if zone:
         vm['zones'] = zone
+    if count:
```

Comment:
any defensive check on --count not able to co-present with other subresource params?

Reply:
I plan to add checks in validator.

Reply:
added